model/stochastic_processes.py

```diff
@@ -3,19 +3,20 @@
 """
 
 import numpy as np
-import experiments.simulation_configuration as simulation_configuration
-from model.constants import blocktime_seconds
 from stochastic import processes
+
+from experiments import simulation_configuration
 from data import historical_values
-from experiments.utils import rng_generator
+from model.constants import blocktime_seconds
+from model.utils.rng_provider import rngp
 
 
 def create_cusd_demand_process(
     timesteps=simulation_configuration.TIMESTEPS,
     dt=simulation_configuration.BLOCKS_PER_TIMESTEP,
     initial_cusd_demand=historical_values.cusd_supply_mean,
     cusd_demand_returns_vola_annually=historical_values.cusd_supply_returns_vola_annually,
-    rng=np.random.default_rng(1)
+    rng=np.random.default_rng()
```

Comment:
Does it make sense to declare these as default values? Or enforce explicitly calling them when creating cusd demand process realizations like its already partly done lower in the file in line 56

Maybe it's better to let it fail when the parameters are not specified in the function call